bookshelf/src/app.php

```diff
@@ -41,7 +41,8 @@
 // turn debug on by default
 $app['debug'] = !in_array(
     getenv('BOOKSHELF_DEBUG'),
-    ['false', '', '0', 'off', 'no']
+    ['false', '', '0', 'off', 'no'],
+    true
```

Comment:
Yeah I agree, it'd be better to guide developers with our logging/debugging tools.